server/src/main/java/io/crate/planner/node/ddl/AlterTableAddColumnPlan.java

```diff
@@ -106,7 +106,6 @@ public void executeOrFail(DependencyCarrier dependencies,
             // Those columns can have overlapping paths, for example we can add columns o['a']['b'] and o['a']['c'].
             // For every added column AnalyzedColumnDefinition provides not only leaf but also path to the root.
             // For o['a']['b'] and o['a']['c'] we can end up having Ref(Ident(o)) and Ref(Ident(a)) twice.
-            // While it doesn't hurt TransportAction since we merge maps there we can stream more Refs than required.
```

Comment:
this is true but no need to mention in comments, sending duplicates is obviously redundant anyway